spec/controllers/descriptions_controller_spec.rb

```diff
@@ -3,44 +3,25 @@
 describe DescriptionsController do
   render_views
 
-  let!(:chart)   { FactoryBot.create :output_element_with_description}
-  let!(:description)   { chart.description}
-
-  describe "#show" do
-    it "should show the description template" do
-      get :show, params: { id: description.id }
-
-      expect(response).to be_successful
-      expect(response).to render_template(:show)
-
-      expect(response.body).to_not be_empty
-    end
-
-    it 'renders nothing if no description exists' do
-      get :show, params: { id: -1 }
-
-      expect(response).to be_successful
-      expect(response.body).to be_empty
-    end
-  end
```

Comment:
Why did these tests dissapear? Is this behaviour no longer required?

Reply:
The show_description route is no longer used in the code base. I removed it along with its specs.